boards/qualcomm/qcc744m_evk/doc/index.rst

```diff
@@ -0,0 +1,102 @@
+.. zephyr:board:: qcc744m_evk
+
+Overview
+********
+
+The QCC744M Evaluation board features the QCC744M module, Which contains a QCC744-2 SoC with 4MB of
```

Comment:
We might get a typo fix PR for it so might as well fix it there `Which` -> `which`

Reply:
done